orb_models/forcefield/gns.py

```diff
@@ -136,6 +149,7 @@ def __init__(
         num_edge_out: int,
         num_mlp_layers: int,
         mlp_hidden_dim: int,
+        kwargs: Dict[str, Any] = None,
```

Comment:
Actually, this class can be removed as I don't think we use it!

Reply:
I added this just in case people pass the attention related kwargs to the non-attention layers, but for second thought it would be good to have an error when this happens.